### PR TITLE
Pathing refactoring

### PR DESCRIPTION
**Pull request by orkaboy**

### CODE REVIEW

**Comment by orkaboy** [comment]
Run has been verified from start of game to end of Luca.

**Comment by orkaboy** [comment]
Run verified up to end of Djose.

**Comment by orkaboy** [comment]
Run verified up to beginning of Macalania Temple.

File: paths/boats.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        # Start Lulu/Wakka conversation
+        6: [0, 0],
+        7: [-43, -17],
+        8: [-34, 12],
+        9: [-23, 85],
+        10: [0, 152],
+        11: [0, 152],
+        12: [22, 104],
+    }
+    checkpoint_fallback = {}
```
Comment (jlward) [inline paths/boats.py:36]: We should add this to the base class. So we don't need to add the empty one in a bunch of places

Comment (orkaboy) [reply paths/boats.py:36]: True.

**Comment by orkaboy** [comment]
Run verified up to beginning of Calm Lands.

File: pathing.py [hunk 2]
```diff
@@ -8,6 +8,15 @@
 FFXC = xbox.controller_handle()
 
 
+class AreaMovementBase:
```
[the remark this answers is not in the document]
Comment (orkaboy) [reply pathing.py]: I fixed this (see `paths/base.py`)

**Comment by orkaboy** [comment]
Run verified to the end of the game.

**Review by jlward — APPROVED** [review]

**Pull request merged**